shared/crypto/libp2p.go

```diff
@@ -0,0 +1,23 @@
+package crypto
+
+import (
+	"encoding/hex"
+
+	"github.com/libp2p/go-libp2p/core/crypto"
+)
+
+// NewLibP2PPrivateKey converts a hex-encoded ed25519d key
+// string into a libp2p compatible Private Key.
+func NewLibP2PPrivateKey(hexString string) (crypto.PrivKey, error) {
```

Comment:
In the identity package, you use `PoktX` as a prefix, but I think we should continue using `LibP2P` where appropriate and avoid using `Pokt` or `Pocket` unless we need to. It's implicit.

Reply:
I understand where you're coming from. My thinking was that 1) things which are currently converting between pocket and libp2p the analogues will be removed as we sort out the future of identity (because we shouldn't need them anymore) and 2) the libp2p based objects will become (or be lightly wrapped by - in future iterations -) the new "implicitly pocket" versions. **I'm getting onboard with perspective though**, I think it's more readable in the interim and I trust that you have the bigger picture in mind. :wink: